common/test/acceptance/tests/lms/test_lms_dashboard.py

```diff
@@ -29,26 +29,63 @@ def setUp(self):
         self.dashboard_page = DashboardPage(self.browser)
 
         # Configure some aspects of the test course and install the settings into the course
-        self.course_fixture = CourseFixture(
-            self.course_info["org"],
-            self.course_info["number"],
-            self.course_info["run"],
-            self.course_info["display_name"],
-        )
-        self.course_fixture.add_advanced_settings({
-            u"social_sharing_url": {u"value": "http://custom/course/url"}
-        })
-        self.course_fixture.install()
-
-        # Create the test user, register them for the course, and authenticate
+        self.courses = {
+            'A': {
+                'org': 'test_org',
+                'number': self.unique_id,
```

Comment:
I copied this example from the search test which also created multiple courses. This actually works. If you watch the browser window as it's running, the test user is subscribed to three courses as they show up on the dashboard. If you `print` it, it prints a different string of numbers.